Makefile

```diff
@@ -8,9 +8,9 @@
 ##         #     (see LICENSE file for the text of the license)         ##
 ##########################################################################
 
-# The default build system is make-based one.
-ifndef COQ_USE_DUNE
-include Makefile.make
-else
+# The default build system is dune-based one.
+ifndef COQ_USE_MAKE
```

Comment:
For who is this variable intended? ie why not do `git mv Makefile.dune Makefile` and tell people "if you want the legacy build system use `make -f Makefile.make`"?

Reply:
Removed, indeed there is not point IMHO.